resources/views/Widgets/Common/ImageBoxWidget.twig

```diff
@@ -85,18 +95,22 @@
     {{ Twig.if("imageUrl | trim is not empty") }}
         {% if widget.settings.lazyLoading.mobile %}
             <lazy-img
-                class="img-{{ imageSize }}"
+                sizing-class="img-{{ imageSize }}"
                 image-url="{{ Twig.print("imageUrl") }}"
+                fallback-url="{{ Twig.print("fallbackImageUrl") }}"
                 alt="{{ Twig.print("imageAltText") }}"
                 {{ Twig.if("imageName | trim is not empty") }}title="{{ Twig.print("imageName") }}"{{ Twig.endif() }}>
             </lazy-img>
         {% else %}
-            <img
-                class="img-{{ imageSize }}"
-                src="{{ Twig.print("imageUrl") }}"
-                alt="{{ Twig.print("imageAltText") }}"
-                {{ Twig.if("imageName | trim is not empty") }}title="{{ Twig.print("imageName") }}"{{ Twig.endif() }}
-            />
+            <picture>
+                <source srcset="{{ Twig.print("imageUrl") }}" {{ Twig.if("'.webp' in imageUrl") }} type="image/webp" {{ Twig.endif() }}>
```

Comment:
Auch hier ist die Prüfung, ob ".webp" in der URL enthalten ist, sehr vage. Auch hier kann der Regex aus der Vue-Komponente verwendet werden. Zum Testen auf reguläre ausdrücke in Twig: https://twig.symfony.com/doc/1.x/templates.html#comparisons